webapp/src/component/layout/TopBar/Notifications.tsx

```diff
@@ -40,9 +40,31 @@ const ListItemHeader = styled(ListItem)`
 `;
 
 export const Notifications: FunctionComponent<{ className?: string }> = () => {
+  const notificationsLoadable = useApiQuery({
```

Comment:
It would be good idea to pass the initial notifications count in the initial data and then create some mechanism to poll it periodically like every 20 minutes or when the user reconnects, so we always get the values accurate.

We're trying to keep the amount of data loaded on the page loaded minimal, so we add the initial reuqirements to the initial data endpoint.

Reply:
Discussed in person - the next PR will add extended support for websockets which will handle the "polling" for updates.